minio/docker-compose.yml

```
@@ -0,0 +1,22 @@
+version: "3.7"
+
+services:
+    app_proxy:
+        environment:
+            APP_HOST: minio_web_1
+            APP_PORT: 9011
+            PROXY_AUTH_ADD: "false"
+
+    web:
+        image: "quay.io/minio/minio:RELEASE.2024-02-17T01-15-57Z"
+        restart: on-failure
+        stop_grace_period: 1m
+        ports:
+            - "9010:9000"
+        command: ["server", "/data"]
+        volumes:
+            - ${APP_DATA_DIR}/data:/data
```

To future proof this let's bind `/data` to the host at
```
${APP_DATA_DIR}/data/minio
``` 
in case other containers get added to this project in the future that require their own data directories.

```suggestion
        volumes:
            - ${APP_DATA_DIR}/data/minio:/data
```

One thing we need to do is commit the data directory to this repo so that it gets installed with correct permissions on a users Umbrel. If we don't do this then Docker will create the directories with root permissions. What you can do is create a `data` directory in the minio app directory, and then inside the `data` directory create a `minio` directory with a `.gitkeep` file in it to make sure it is committed even though it is an empty directory.

Take a look at Gitea as an example: https://github.com/getumbrel/umbrel-apps/tree/master/gitea